src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -33,6 +37,20 @@ public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHea
         switch (wrapper.getState()) {
             case CREATE_HEARING:
                 createHearing(wrapper);
+                HearingRequestPayload hearingRequestPayload = new HearingRequestPayload();
+                HearingResponse response =
+                    hearingApi.createHearingRequest("authorisation",
+                                                    "serviceAuthorization",
+                                                    hearingRequestPayload);
+                //Fake payload for now. Ask Lucas about differences in mapping objects.
+                //check which case data to use Ask Lucas
+                wrapper.getUpdatedCaseData().setHearingID(String.valueOf(response.getHearingRequestId()));
```

Comment:
Use addHearingResponse for POST response, i.e. create hearing.
updateHearingResponse is for put and delete, with the delete one using the overloaded method with  cancellationReasonCode